src/Erdene/Divide_two_num_without_div_operator.java

```diff
@@ -0,0 +1,24 @@
+package Erdene;
+
+public class Divide_two_num_without_div_operator {
+    public static void Divide(double Dividend, double divisor) {
+        int count = 1;
```

Comment:
int count should be 0 at the beginning, when I put (10, 3) gives me 4

Reply:
I pushed my update, please consider